src/store.c

```diff
@@ -252,9 +292,10 @@ static bool writer_open(
 
     size_t len =
         sizeof(struct header) +
-        vals_cap(vals) +
+        indexer_cap(pairs) +
+        indexer_cap(pairs) +
```

Comment:
consider `(2 * indexer_cap(pairs))` or add a small comment that says column a and column b

Reply:
So as I mentioned last week, the capacity of the index shouldn't be computed on the number of pairs but on the numbers values for that column as the index will contain an entry per value and per pair.

Reply:
Working on this, thanks!

Reply:
This was fixed a while ago. Thanks.